cluster/manifests/skipper/deployment.yaml

```diff
@@ -202,6 +202,9 @@ spec:
           - "-write-timeout-server={{ .ConfigItems.skipper_write_timeout_server }}"
           - '-default-filters-prepend={{ .ConfigItems.skipper_default_filters }}'
           - "-disabled-filters={{ .ConfigItems.skipper_disabled_filters }}"
+{{ if ne .ConfigItems.skipper_edit_route_placeholders "" }}
+          - '-edit-route={{ .ConfigItems.skipper_edit_route_placeholders }}'
```

Comment:
Multiple values require https://github.com/zalando/skipper/issues/2106

Reply:
will this fail in case `skipper_edit_route_placeholders` is not defined?

Reply:
I've added default empty value for it into config-defaults.yaml